sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -48,16 +49,17 @@
  * @see EventHubProducerClient
  * @see EventHubProducerAsyncClient
  */
-public class EventData {
+public class EventData extends MessageContent {
```

Comment:
Add a note in changelog as it is a [potentially breaking change](https://revapi.org/revapi-java/0.26.1/differences.html#java.class.nonFinalClassInheritsFromNewClass)